code/game/gamemodes/clock_cult/clock_structures.dm

```diff
@@ -427,10 +427,10 @@
 /obj/effect/clockwork/judicial_marker/New()
 	..()
 	flick("judicial_marker", src)
-	spawn(25) //Utilizes spawns due to how it works with Ratvar's flame
+	spawn(16) //Utilizes spawns due to how it works with Ratvar's flame
```

Comment:
I dislike trying to sync sprites with things that update slower than the sprites, so I'd really rather not, especially if it turns out I have to change this again.